test/index.js

```diff
@@ -105,6 +105,22 @@ test('test a react component with custom name function', function(t) {
   t.equal(output, '<Basic test1={_testCallBack1}\n  test2={_testCallBack2} />');
 });
 
+test('test a react component with autodetection of the functions names', function(t) {
+  t.plan(1);
+
+  let _testCallBack1 = function () {
+    //no-op
+  };
+
+  let output = jsxToString(
+    <Basic test1={_testCallBack1} test2={function() {}} />, {
```

Comment:
why an `anonymous` function is being converted to `test2` name?

Also, depending on the version of NodeJs you have, different names will yield for anonymous functions.

Check out this Jest issue where they were using the function names before, and decided to stop using it due to all the side-effects it caused.

https://github.com/facebook/jest/issues/1681
https://github.com/facebook/jest/pull/1717

I'm not sure how the function name would help in the `jsx-to-string`, if there is a way to provide the implementation for the function, then I can definitely see value, but IMHO just the name does not add much, this is why I'm using `...`.